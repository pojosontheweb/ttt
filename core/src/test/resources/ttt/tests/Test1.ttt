<%!
    Foo foo;
%>
this is a <%= foo.getBar() %> template