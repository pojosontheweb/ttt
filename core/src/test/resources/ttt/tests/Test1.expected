package ttt.tests;

public class Test1 extends com.pojosontheweb.ttt.Template {

	private final Foo foo;

	public Test1(Foo foo) {
		this.foo = foo;
	}

	@Override
	public void render(java.io.Writer out) throws java.io.IOException {
		write(out, "this is a " );
		write(out,  foo.getBar()  );
		write(out, " template" );
	}
}
